reconcilers/reconcilers.go

```diff
@@ -981,17 +995,16 @@ var (
 
 // ChildReconciler is a sub reconciler that manages a single child resource for a reconciled
 // resource. The reconciler will ensure that exactly one child will match the desired state by:
-// - creating a child if none exists
-// - updating an existing child
-// - removing an unneeded child
-// - removing extra children
+//   - creating a child if none exists
+//   - updating an existing child
+//   - removing an unneeded child
+//   - removing extra children
 //
 // The flow for each reconciliation request is:
-// - DesiredChild
-// - if child is desired:
-//    - HarmonizeImmutableFields (optional)
-//    - MergeBeforeUpdate
-// - ReflectChildStatusOnParent
+//   - DesiredChild
+//   - if child is desired, HarmonizeImmutableFields (optional)
```

Comment:
godoc now supports lists, but not nested lists

Ref https://tip.golang.org/doc/comment#lists